### ISSUE TITLE
Conversion 'py2rpy' not defined for objects of type '<class 'anndata.compat._overloaded_dict.OverloadedDict'>'

### ISSUE THREAD

**Issue by gokceneraslan** [theislab/anndata2ri#46]
When I run [this code](https://github.com/normjam/benchmark/blob/master/normbench/methods/ad2seurat.py) using an AnnData object I get the following error. Any ideas about the traceback?

```pytb
---------------------------------------------------------------------------
NotImplementedError                       Traceback (most recent call last)
<ipython-input-7-076c4f95bd62> in <module>
----> 1 sct = pyScTransform(kera)

<ipython-input-6-b221c7997319> in pyScTransform(adata, output_file)
     19                 adata.layers[key].sort_indices()
     20 
---> 21     ro.globalenv['adata'] = adata
     22     ro.r('seurat_obj = as.Seurat(adata, counts="X", data = NULL)')
     23     ro.r('res <- SCTransform(object=seurat_obj, return.only.var.genes = TRUE, do.correct.umi = FALSE)')

/opt/conda/lib/python3.7/site-packages/rpy2/robjects/environments.py in __setitem__(self, item, value)
     30 
     31     def __setitem__(self, item: str, value: typing.Any) -> None:
---> 32         robj = conversion.converter.py2rpy(value)
     33         super(Environment, self).__setitem__(item, robj)
     34 

/opt/conda/lib/python3.7/functools.py in wrapper(*args, **kw)
    838                             '1 positional argument')
    839 
--> 840         return dispatch(args[0].__class__)(*args, **kw)
    841 
    842     funcname = getattr(func, '__name__', 'singledispatch function')

/opt/conda/lib/python3.7/site-packages/anndata2ri/py2r.py in py2rpy_anndata(obj)
     69         # Convert everything we know
     70         with localconverter(full_converter() + dict_converter):
---> 71             metadata = ListVector(obj.uns.items())
     72 
     73         rd_args = {conv_name.scanpy2sce(k): mat_converter.py2rpy(obj.obsm[k]) for k in obj.obsm.keys()}

/opt/conda/lib/python3.7/site-packages/rpy2/robjects/vectors.py in __init__(self, tlist)
    639                 raise ValueError('tlist should have a /method/ __iter__ '
    640                                  '(not an attribute)')
--> 641             kv = [(str(k), conversion.py2rpy(v)) for k, v in tlist]
    642             kv = tuple(kv)
    643             df = baseenv_ri.find("list").rcall(kv, globalenv_ri)

/opt/conda/lib/python3.7/site-packages/rpy2/robjects/vectors.py in <listcomp>(.0)
    639                 raise ValueError('tlist should have a /method/ __iter__ '
    640                                  '(not an attribute)')
--> 641             kv = [(str(k), conversion.py2rpy(v)) for k, v in tlist]
    642             kv = tuple(kv)
    643             df = baseenv_ri.find("list").rcall(kv, globalenv_ri)

/opt/conda/lib/python3.7/functools.py in wrapper(*args, **kw)
    838                             '1 positional argument')
    839 
--> 840         return dispatch(args[0].__class__)(*args, **kw)
    841 
    842     funcname = getattr(func, '__name__', 'singledispatch function')

/opt/conda/lib/python3.7/site-packages/rpy2/robjects/conversion.py in _py2rpy(obj)
    172     raise NotImplementedError(
    173         "Conversion 'py2rpy' not defined for objects of type '%s'" %
--> 174         str(type(obj))
    175     )
    176 

NotImplementedError: Conversion 'py2rpy' not defined for objects of type '<class 'anndata.compat._overloaded_dict.OverloadedDict'>'
```

**Comment by gokceneraslan** [comment]
anndata version: '0.7.3'

**Comment by nkanrar** [comment]
I am getting the same error. I have anndata version 0.7.4, anndata2ri version 1.0.2, get_version 2.1, and tzlocal version 2.0.0. Not sure what the issue is. Would appreciate any help on sorting this out.

**Comment by nkanrar** [comment]
I was combing through some of the documentation, and it seems like this is an issue that comes up because the AnnData to SingleCellExperiment conversion has issues with the 'neighbors' parameter in adata.uns. I'm referring to this thread https://github.com/theislab/anndata/pull/337 which does discuss that there are issues with storing the adata.uns['neighbors'] dictionary in the SingleCellExperiment object. 

What works for me right now is just saving the neighbors dictionary separately, deleting the 'neighbors' key from adata.uns, and then adding it back once I go back to the AnnData object.

**Comment by flying-sheep** [comment]
released in 1.0.3!

**Comment by JakeLehle** [comment]
Had the same issue and updated to anndata2ri 1.0.3 to fix it. You will get a warning that objects that have no class will not be converted over.
